projects/aca-content/src/lib/components/search/search-input/search-input.component.html

```diff
@@ -7,13 +7,13 @@
   <button mat-icon-button class="app-search-button" (click)="searchByOption()" [title]="'SEARCH.BUTTON.TOOLTIP' | translate">
     <mat-icon [attr.aria-label]="'SEARCH.BUTTON.ARIA-LABEL' | translate">search</mat-icon>
   </button>
-  <mat-form-field class="app-input-form-field" [floatLabel]="'auto'">
+  <mat-form-field class="app-input-form-field app-input-form-field-disabled" [floatLabel]="'auto'">
```

Comment:
Maybe we can rename the class to `app-input-form-field-readonly` instead to reflect the correct param?